DscResources/NTFSAccessEntry/NTFSAccessEntry.psm1

```diff
@@ -669,53 +615,74 @@ Function Get-MappedGenericRight
     (
         [Parameter(Mandatory = $true)]
         [int]
-        $Rights
+        $rights
```

Comment:
This one was actually correct. $Rights

Reply:
I'll address this and another thought I had in the Test- function I wrote at the end, I'll make this change and the other...